input/resources/codesystem-au-recorded-sex-or-gender-source-document-type.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="au-recorded-sex-or-gender-source-document-type" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/CodeSystem/au-recorded-sex-or-gender-source-document-type" />
+    <identifier>
+        <system value="urn:ietf:rfc:3986"/>
+        <value value="urn:oid:2.16.840.1.113883.2.3.4.1.4.30"/>
+    </identifier>
+    <name value="AURecordedSexOrGenderSourceDocumentType" />
+    <title value="AU Recorded Sex or Gender Source Document Type" />
+    <status value="active" />
+    <experimental value="false"/>
+    <description value="Codes that represent source document types associated with a Recorded Sex or Gender (RSG) value in an Australian context." />
+    <caseSensitive value="true" />
+    <hierarchyMeaning value="is-a"/>
+    <valueSet value="http://terminology.hl7.org.au/ValueSet/au-recorded-sex-or-gender-source-document-type"/>
+    <compositional value="false" />
+    <versionNeeded value="false" />
+    <content value="complete" /> 
+    <concept>
+      <code value="passport"/>
+      <display value="Passport"/>
+      <definition value="A passport issued by a jurisdiction"/>
```

Comment:
Most definitions are missing punctuation. Add full stop add end of definition text where missing.

Reply:
Done.